include/dsCompositeIn.h

```diff
@@ -252,6 +253,21 @@ dsError_t dsCompositeInScaleVideo (int32_t x, int32_t y, int32_t width, int32_t
  */
 typedef void (*dsCompositeInConnectCB_t)(dsCompositeInPort_t Port, bool isPortConnected);
 
+/**
+ * @brief Notifies applications when the Composite input port video mode changes
+ *
+ * HAL Implementation must call this method to deliver updated Composite input port video mode info
```

Comment:
Does the below info provides better info
"The HAL implementation must call this method to provide the caller with updated video mode information for the composite input port whenever there is a change in the received video mode."